apps/ai-image-tagging/lambda/usage.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+
+const { DateTime } = require('luxon');
+
+const documentClient = new AWS.DynamoDB.DocumentClient();
+
+module.exports = async spaceId => {
+  const now = new Date();
+  const dt = DateTime.fromObject({
```

Comment:
```suggestion
  const period = DateTime.utc().startOf('month').toSeconds()
```

does everything from line 10 to line 17